src/Commands/SetGearRoller.cpp

```diff
@@ -23,6 +24,9 @@ bool SetGearRoller::IsFinished() {
   if (rollerSpeedRPM > 0)
     return GearPickup::getInstance()->IsGearIn();
   else if (rollerSpeedRPM < 0)
+    if(timeout.HasPeriodPassed(2))
+      return true;
+    else
```

Comment:
This is a failsafe for GearRoller. The GearRoller will only run backwards for 2 seconds. This is how eject works. This is fine for now. Seems like this should be done differently.